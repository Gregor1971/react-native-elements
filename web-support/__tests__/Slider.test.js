import React from 'react';
import { Text, TouchableHighlight, View } from 'react-native';
import parseSamples, {
  assignRef,
  genRefId,
} from 'enzyme-styleguidist-sample-parser';
import options, {
  snapShot,
  buildJsxForGuideMethod,
  ensureCalled,
  onlyEnsureCalled,
  onlySnapshots,
} from '../samples';
import { Slider as Component } from '../../src';

const props = {
  'no props': {
    component: Component,
    enzyme: {
      tests: { shallow: { snapshot: snapShot() } },
    },
    styleguidist: {},
  },
};

const samples = {
  props,
};

parseSamples(
  {
    Input: { sectionComponents: { Slider: { samples } } },
  },
  options
);
