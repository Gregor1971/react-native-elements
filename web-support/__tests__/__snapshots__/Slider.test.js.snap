// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input: Slider: [ prop: no props ] enzyme-shallow: snapshot 1`] = `
<View
  onLayout={[Function]}
  step={0}
  style={
    Array [
      106,
      undefined,
    ]
  }
  value={0}
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "#b3b3b3",
        },
        107,
        undefined,
      ]
    }
  />
  <AnimatedComponent
    style={
      Array [
        107,
        undefined,
        Object {
          "backgroundColor": "#3f3f3f",
          "opacity": 0,
          "position": "absolute",
          "width": AnimatedAddition {
            "_a": AnimatedInterpolation {
              "_children": Array [],
              "_interpolation": [Function],
              "_listeners": Object {},
              "_parent": AnimatedValue {
                "_animation": null,
                "_children": Array [],
                "_listeners": Object {},
                "_offset": 0,
                "_value": 0,
              },
            },
            "_b": AnimatedValue {
              "_animation": null,
              "_children": Array [],
              "_listeners": Object {},
              "_offset": 0,
              "_value": 0,
            },
            "_children": Array [],
            "_listeners": Object {},
          },
        },
      ]
    }
  />
  <AnimatedComponent
    onLayout={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "red",
        },
        108,
        undefined,
        Object {
          "opacity": 0,
          "transform": Array [
            Object {
              "translateX": AnimatedInterpolation {
                "_children": Array [],
                "_interpolation": [Function],
                "_listeners": Object {},
                "_parent": AnimatedValue {
                  "_animation": null,
                  "_children": Array [],
                  "_listeners": Object {},
                  "_offset": 0,
                  "_value": 0,
                },
              },
            },
            Object {
              "translateY": -0,
            },
          ],
        },
      ]
    }
  />
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        109,
        Object {},
      ]
    }
  />
</View>
`;
